sycl/test-e2e/ESIMD/spec_const/spec_const_redefine_esimd.cpp

```diff
@@ -0,0 +1,102 @@
+// REQUIRES: gpu
+// UNSUPPORTED: gpu-intel-gen9 && windows
+// UNSUPPORTED: cuda || hip
```

Comment:
If I remember it correctly, SPEC consts do not work with esimd_emulator:
```suggestion
// UNSUPPORTED: cuda || hip || esimd_emulator
```

Reply:
done thanks